packages/valory/skills/termination_abci/tests/test_behaviours.py

```diff
@@ -488,6 +506,26 @@ def _mock_get_tx_data_request(
                 err_reqs=[],
                 expected_logs=[_SUCCESS_LOG],
             ),
+            BehaviourTestCase(
+                name="agent drops message because app already stopped",
+                initial_data=_INITIAL_DATA,
+                ok_reqs=[],
+                err_reqs=[
+                    _mock_is_stopped,
+                    _mock_get_service_owner_request,
+                ],
+                expected_logs=[_IS_STOPPED_LOG],
+            ),
+            BehaviourTestCase(
+                name="agent could not send message because state != WAITING_MESSAGE",
+                initial_data=_INITIAL_DATA,
+                ok_reqs=[],
+                err_reqs=[
+                    _mock_state_is_not_waiting_message,
+                    _mock_get_service_owner_request,
+                ],
+                expected_logs=[_IS_NOT_WAITING_MESSAGE],
+            ),
```

Comment:
@0xArdi is this acceptable? These first of these `err_reqs` here are plain mocks, not request mocks. I didn't want to change the `BehaviourTestCase` with additional "pre-mocks". Also the `_mock_get_service_owner_request` is not truly an error mock, just need a mock that will trigger the callback, but not trigger `.complete()`.

Reply:
Yeah I think its still clear what is happening. We could think of a better name than ok_reqs, err_reqs for the future.